caluma/tests/snapshots/snap_test_schema.py

```diff
@@ -39,8 +39,8 @@
 
 interface Answer {
   id: ID
-  created: DateTime!
-  modified: DateTime!
+  createdAt: DateTime!
+  modifiedAt: DateTime!
```

Comment:
Where is `createdByUser` and `createdByGroup`?

Reply:
Very good point. I missed adding it to the interfaces. Changed now.